src/pkg/leaderElection.go

```diff
@@ -43,8 +44,26 @@ func RunLeaderElection(client *clientset.Clientset, lockName, lockIdentity, lock
 				log.Info().Msgf("Perform Migration")
 				for {
 					log.Info().Msgf("leader is %s", lockIdentity)
-					log.Info().Msgf("Getting replica count...")
-					time.Sleep(5 * time.Second)
+					replicaCount, err := getReplicas()
+					if err != nil {
+						log.Fatal().Msgf("Failed to get replica count")
+					}
+					log.Info().Msgf("Setting replica count to %v", replicaCount)
+					retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
```

Comment:
Going to see if I can pass a func here but was getting a type error.

Reply:
whats the point of this retry?  Since we are in a infinite for loop - if we get a error we'd just come back around again.

Reply:
This was recommended from client-go example:  https://github.com/kubernetes/client-go/blob/master/examples/create-update-delete-deployment/main.go#L117
My understanding is this wouldn't overwrite other updates but could be wrong. Basically just pulled from their example though.

Reply:
interesting.  Ok that makes sense.  We should link to this in a comment for future selfs